admin/options_values_manager.php

```diff
@@ -835,7 +835,7 @@
                     ?>
                     <div class="col-md-4">
                       <div class="form-group">
-                        <?php echo zen_draw_pull_down_menu('option_id', $optionsValueArray, '', 'class="form-control"'); ?>
+                        <?php echo zen_draw_pull_down_menu('option_id', $optionsValueArray, $filter, 'class="form-control" title="' . __LINE__ . '"'); ?>
```

Comment:
> `title="' . __LINE__ . '`

I'm assuming you didn't mean to leave this in here.

Reply:
correct.